src/zcash/ZcashTools.ts

```diff
@@ -33,9 +30,7 @@ export class ZcashTools implements EdgeCurrencyTools {
   currencyInfo: EdgeCurrencyInfo
   io: EdgeIo
   networkInfo: ZcashNetworkInfo
-
-  KeyTool: typeof KeyToolType
-  AddressTool: typeof AddressToolType
+  Tools: typeof ToolsType
```

Comment:
Can we call this `nativeTools` or such? Capitalized names are always classes or types, and lowercase `tools` usually means `EdgeCurrencyTools` in this repo.